src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -938,9 +938,10 @@ bool ResumeCtrlImpl::CheckIgnCyclesData() const {
   LOG4CXX_AUTO_TRACE(logger_);
   const uint32_t global_ign_on_count =
       resumption_storage_->GetGlobalIgnOnCounter();
-  const uint32_t allowed_ign_on_value = 2;
+  const uint32_t the_first_ignition = 1;
   const bool is_emergency_ign_off_occurred =
-      global_ign_on_count >= allowed_ign_on_value;
+      global_ign_on_count > the_first_ignition;
+  // In case of resumption it should be minimum second ignition cycle.
```

Comment:
This comment does not match the functionality. This count is reset on `IGNITION_OFF`, so a value greater than `1` means that the project was shut down by force. A value of `1` does not meant that it is the first ignition cycle.

Reply:
@jacobkeeler fixed 06ce5d9b935a3eaaaac5b08686bb50a7d4ae8eec